src/components/SpiderDiagram/SpiderDiagram.tsx

```diff
@@ -58,6 +58,7 @@ export const SpiderDiagram = ({
 
   const offsetX = 0;
 
+  /* eslint-disable sonarjs/todo-tag */
   // TODO: add some screenreader capability?
```

Comment:
I will create a separate JIRA ticket for this accessibility case. This component really needs some work on that.

Reply:
Ticket is created: https://jira.eduuni.fi/browse/OPHJOD-1213